starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,46 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+h1{
+  color: white;
+  background-image: url('/Users/Kristen/Desktop/lab-css-recipes-clone/starter-code/french-onion-soup-starter-code/images/soup.jpg');
```

Comment:
to link your image to the folder its located in your would set the `url('../images/soup.jpg')`